DigitalLearningSolutions.Web/Views/Shared/Components/ContentsList/Default.cshtml

```diff
@@ -0,0 +1,21 @@
+@using DigitalLearningSolutions.Web.ViewModels.Common.MiniHub

```

Comment:
Would it be worth putting the Mini-Hub related shared components into a separate minihub foldeR?

Reply:
Yeah, you're probably right.